Week-2/index2-1.html

```diff
@@ -0,0 +1,359 @@
+
+        </section>
+        <section class="section-inside">
+            <article>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</article>
+
+            <article>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</article>
+            <article>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</article>
+
+        </section>
+    </div>
+    <div class="form-bg">
+        <form>
+
+            <h2 class="form-heading">Get Started</h2>
+        <div class="form-inside">
+        <div class="inside-1">
+            <div>
+                <h5>Tittle</h5>
+            <label class="container">Mr
+                <input type="radio" name="radio" checked>
+              
+              </label>
+              <label class="container">Mrs
+                <input type="radio" name="radio">
+                
+              </label>
+              <label class="container">Miss
+                <input type="radio" name="radio">
+              
+              </label>
+            
+            </div>
+           <div class="form-div">
+            <label>Name</label>
+            <input type="text" required>
+           </div>
+           <div class="form-div">
+            <label>Last Name</label>
+            <input type="text" required>

```

Comment:
some fileds might not be required, depends on the use case

Reply:
ok mam i will check